htseq.wdl

```diff
@@ -25,38 +25,39 @@ task HTSeqCount {
         Array[File]+ inputBams
         File gtfFile
         String outputTable = "output.tsv"
-        String format = "bam"
         String order = "pos"
         String stranded = "no"
         String? featureType
         String? idattr
         Array[String] additionalAttributes = []
 
+        Int nprocesses = 1
         String memory = "8G"
         Int timeMinutes = 10 + ceil(size(inputBams, "G") * 60)
-        String dockerImage = "quay.io/biocontainers/htseq:0.11.2--py37h637b7d7_1"
+        String dockerImage = "quay.io/biocontainers/htseq:0.12.4--py37hb3f55d8_0"
     }
 
     command {
         set -e
         mkdir -p "$(dirname ~{outputTable})"
         htseq-count \
-        -f ~{format} \
```

Comment:
Should this change also be noted in the CHANGELOG?

Reply:
Yes it should.